tests/ztimer_systick/Makefile

```diff
@@ -0,0 +1,6 @@
+include ../Makefile.tests_common
+
+USEMODULE += ztimer_msec ztimer_usec ztimer_sec
```

Comment:
Won't that just pull in the usual timer / RTT backends?
Then `ztimer/periph_systick.c` wouldn't even be used by this test.